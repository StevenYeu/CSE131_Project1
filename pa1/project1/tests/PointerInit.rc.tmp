Error, "project1/tests/PointerInit.rc": 
  Initialization value of type int not assignable to constant/variable of type int*.
Compile: failure.
