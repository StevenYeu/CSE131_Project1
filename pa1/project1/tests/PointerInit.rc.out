Error, "/tmp/cs131_testrunner_RJ0qMF/file.rc": 
  Initialization value of type int not assignable to constant/variable of type int*.
Compile: failure.
